docs/spec/spec.md

```diff
@@ -58,33 +70,49 @@ broker._
+| `config`   | [`duckv1.KReference`](#duckv1.kreference)                               | Optional    | Reference to the configuration options for this broker. For example, this could be a pointer to a ConfigMap. |             |
+| `delivery` | [`eventingduckv1beta1.DeliverySpec`](#eventingduckv1beta1.deliveryspec) | Optional    | The delivery specification for events within the broker mesh. This includes things like retries, DLQ, etc.   |             |
 
 #### Status
 
-| Field              | Type        | Description                                                                                  | Constraints |
-| ------------------ | ----------- | -------------------------------------------------------------------------------------------- | ----------- |
-| observedGeneration | int64       | The 'Generation' of the Broker that was last processed by the controller.                    |             |
-| conditions         | Conditions  | Broker conditions.                                                                           |             |
-| address            | Addressable | Address of the endpoint which meets the [_Addressable_ contract](interfaces.md#addressable). |             |
+The `BrokerStatus` represents the current state of the `Broker`.
+
+| Field Name           | Field Type                                  | Requirement | Description                                                                                                                        | Constraints |
+| -------------------- | ------------------------------------------- | ----------- | ---------------------------------------------------------------------------------------------------------------------------------- | ----------- |
+| `observedGeneration` | `int`                                       | Optional    | The 'Generation' of the `Broker` that was last processed by the controller.                                                        |             |
+| `conditions`         | [`[]apis.Condition`](#apis.condition)       | Optional    | Broker conditions. The latest available observations of the resource's current state.                                              |             |
+| `annotations`        | `map[string]string`                         | Optional    | Fields to save additional state as well as convey more information to the user.                                                    |             |
+| `address`            | [`duckv1.Addressable`](#duckv1.addressable) | Required    | Exposed endpoint (as an URI) for getting events delivered into the broker mesh. The broker is [`Addressable`](duckv1.addressable). |             |
 
 ##### Conditions
 
 - **Ready.** True when the Broker is provisioned and ready to accept events.
-- **Addressable.** True when the Broker has an resolved address in it's status.
+- **IngressReady.** True when broker ingress is ready.
```

Comment:
Here again, we should only require the top level Ready condition. For example, brokers that do not have channel backed implementations, they will not have a TriggerChannelReady condition.